### DISCUSSION TITLE
Cover Shrinks

### DISCUSSION THREAD

**Discussion by utamayo** [GitbookIO/community#720] [0. General bugs and questions]
In my page, the cover you can see perfectly but when it is published in a doc the cover shrinks a little bit so it change quite a lot in depends the situation and the result is not as good as you think.
Here the difference:
![image](https://github.com/user-attachments/assets/a8dd279c-b635-41ba-994c-54b8629138f8)
![image](https://github.com/user-attachments/assets/70f18cb5-4c10-407a-b48a-3bcdfafe7a11)

**Comment by aikoii** [comment]
Hey @utamayo! While we aim for a one-size-fits-all solution, it can be a bit tricky to make every image display perfectly. Some images may not display optimally with the recommended size of 1990x480. Using an image closer to that aspect ratio should give you a better result 🤞

**Comment by 0xBraunGuy** [comment]
It would be helpful to add this info directly in the UI when uploading / changing the cover, I also spent a lot of time trying to figure out how to get the cover to display correctly
